services/roomService/src/client/TestUtils.ts

```diff
@@ -94,17 +95,28 @@ export function setSessionTokenAndTownID(coveyTownID: string, sessionToken: stri
 
 export function createMessageForTesting(
   type: MessageType,
-  player1Id: string,
-  player2Id?: string,
+  player1: Player,
+  player2?: Player,
 ): Message {
   const timestamp = Date.now().toString();
   let directMessageID;
-  if (player2Id) {
-    directMessageID = `${player1Id}:${player2Id}`;
+  if (player2) {
+    directMessageID = `${player1.id}:${player2.id}`;
+  }
+  if (type == MessageType.ProximityMessage){
+    return {
+      userName: nanoid(),
+      userId: player1.id,
+      location: player1.location,
+      messageContent: "Omg I'm a test",
+      timestamp,
+      type,
+      directMessageId: directMessageID,
```

Comment:
we shouldn't be setting the `directMessageId` unless it's a direct message, best to leave it out until we hit that case. we don't even need to set the `let` above unless it's a dm

Reply:
sounds good! I'll save it for the dm PR